src/python/CRABClient/Commands/submit.py

```diff
@@ -245,10 +249,11 @@ def validateConfig(self):
         ## Load the external plugin or check that the crab plugin is valid.
         external_plugin_name = getattr(self.configuration.JobType, 'externalPluginFile', None)
         crab_plugin_name = getattr(self.configuration.JobType, 'pluginName', None)
-        crab_job_types = {'ANALYSIS': None, 'PRIVATEMC': None, 'COPYCAT': None} #getJobTypes()
+        crab_job_types = {'ANALYSIS': None, 'PRIVATEMC': None, 'COPYCAT': None, 'RECOVER': None} #getJobTypes()
         if external_plugin_name:
             addPlugin(external_plugin_name) # Do we need to do this here?
         if crab_plugin_name:
+            # FIXME we do not have only analysis or MC
```

Comment:
do we really need to submit using different plugin types ? all in all now you support RECOVERY w/o changing this !

Reply:
I see, you have a very good point. 
Indeed, when using copycat and recover jobtype, we should not have anything different from analysis and privatemc after line 107 where we have `plugjobtype.run([...])`. I will remove recover and copycat from line 252, remove this comment, and everything should be fine.
Thanks! :)

Reply:
actually no, this is checked earlier on, triggered by `SubCommand.__init__()`, so we need to specify all the job types.